### PR TITLE
[BD-46] feat: renamed custom media query

### LINKED ISSUE #2444: [alpha] Rename custom media query output to ensure `pgn` namespace is the prefix
The design tokens build process generates the following file: https://github.com/openedx/paragon/blob/alpha/styles/css/core/custom-media-breakpoints.css

It contains `@custom-media` definitions such as:

```css
@custom-media --max-pgn-size-breakpoint-xs (max-width: 576px);
```

However, the `min` and `max` substrings are coming before the `pgn` prefix. 

To ensure consistency across all tokens/variables available to consumers (e.g., `--pgn-color-primary-500`), we should ensure the `@custom-media` query variables include `pgn` as the prefix, e.g.:

```css
@custom-media --pgn-size-breakpoint-min-width-xs (min-width: 0);
@custom-media --pgn-size-breakpoint-max-width-xs (max-width: 576px);
@custom-media --pgn-size-breakpoint-min-width-sm (min-width: 0);
@custom-media --pgn-size-breakpoint-max-width-sm (max-width: 768px);
```

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

Issue: https://github.com/openedx/paragon/issues/2444

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Review by monteri — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: tokens/style-dictionary.js [hunk 1]
```diff
@@ -185,9 +186,9 @@ StyleDictionary.registerFormat({
 
     for (let i = 0; i < breakpoints.length; i++) {
       const [currentBreakpoint, nextBreakpoint] = [breakpoints[i], breakpoints[i + 1]];
-      customMediaVariables += `@custom-media --min-${currentBreakpoint.name} (min-width: ${currentBreakpoint.value});\n`;
+      customMediaVariables += `${composeBreakpointName(currentBreakpoint.name, 'min')} (min-width: ${currentBreakpoint.value});\n`;
```
Comment (adamstankiewicz) [inline tokens/style-dictionary.js:189]: Nice refactor to abstract this out into a helper function!

**Review by adamstankiewicz — APPROVED** [review]
LGTM!

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2495&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-08-03+08:24&entry.1917517419=2023-08-03+18:51&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**